src/commons/workspace/Workspace.tsx

```diff
@@ -222,7 +223,11 @@ const Workspace: React.FC<WorkspaceProps> = props => {
         </Resizable>
         <div className="row content-parent" ref={contentContainerDiv}>
           <div className="editor-divider" ref={editorDividerDiv} />
-          <Resizable {...editorResizableProps()}>{createWorkspaceInput(props)}</Resizable>
+          {isVscode ? (
+            <div style={{ width: '0px' }}>{createWorkspaceInput(props)}</div>
```

Comment:
It must still be visible is it? We can't use `visibility: hidden`? If we can't, then in addition to `width: 0`, should we also use `overflow: hidden`?

Reply:
While `visibility: hidden` wouldn't work, I think we can completely remove it now that messages are passed directly to the redux store rather than to the editor.